shared/utils/build-all-guides.py

```diff
@@ -171,6 +171,17 @@ def generate_guide_for_input_content(input_content, benchmark_id, profile_id):
     return ret
 
 
+def get_cpu_count():
+    import multiprocessing
```

Comment:
Nitpick: I believe that, according to pep8, imports should be at the top of the file and not in functions. Otherwise, LGTM.